client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
+			mapValues( val => `${ val }Z` ),
+			mapKeys( key => key.slice( 0, -'_gmt'.length ) )
```

Comment:
if we want to truly explain why the -1: `-4 /* truncate "_gmt" */`? I don't see any way to give the "truncate" meaning to the `-1` in code?